plugins/exec.py

```diff
@@ -1,16 +1,12 @@
 """
 exec.py: Provides commands for executing raw code and debugging PyLink.
 """
-import importlib
 import pprint
 # These imports are not strictly necessary, but make the following modules
 # easier to access through eval and exec.
-import re
 import threading
-import time
 
-import pylinkirc
-from pylinkirc import *
+from pylinkirc import utils, world
```

Comment:
See the above comment, most of these are convenience imports to make eval / exec more convenient.

Reply:
To make `from pylinkirc import *` work as intended, an `__init__.py` file should be created to define an `__all__`. Which seems to be currently problematic.